### ISSUE TITLE
Failing test 

### ISSUE THREAD

**Issue by matthewbauer** [jaspervdj/stylish-haskell#206]
Hi,

We are getting a failing test for Language.Haskell.Stylish.Step.SimpleAlign.Tests case 08, in Stylish v0.9.1.1:

```
expected: "canDrink mbAge = case mbAge of\n    Just age | age > 18 -> True\n    _                   -> False\n"
 but got: "canDrink mbAge = case mbAge of\n    Just age | age > 18 -> True\n    _        -> False\n"
```

https://hydra.nixos.org/job/nixpkgs/trunk/haskellPackages.stylish-haskell.x86_64-linux
https://hydra.nixos.org/job/nixpkgs/trunk/haskellPackages.stylish-haskell.x86_64-darwin

**Comment by jaspervdj** [comment]
I'm not sure what's happening here, this test was added _after_ I released `v0.9.1.1`.  It is not present in `v0.9.1.1`:

<https://github.com/jaspervdj/stylish-haskell/blob/v0.9.1.1/tests/Language/Haskell/Stylish/Step/SimpleAlign/Tests.hs#L27>

**Comment by k0ral** [comment]
Unless I am mistaken, there is no guarantee that the tag from git matches the tag from hackage. You might simply have run `cabal sdist` and `cabal upload` on a local clone of the repository that had extra changes compared to `0.9.1.1` tag from git (happened to me once).

As a matter of fact, this test was present when the package got uploaded to Hackage: https://hackage.haskell.org/package/stylish-haskell-0.9.1.1/src/tests/Language/Haskell/Stylish/Step/SimpleAlign/Tests.hs

**Comment by jaspervdj** [comment]
Should be fixed in `v0.9.2.0`

**Comment by matthewbauer** [comment]
Thanks!
